specs/AppNotifications/AppNotificationContentSpec/AppNotificationBuilder-spec.md

```diff
@@ -0,0 +1,764 @@
+styles.
+
+```c#
+AppNotificationContent()
+    .SetUseButtonStyle(true)
+    .AddText(Text(L"Content"))
+    .AddTextBox(TextBox(L"textBox")
+        .SetPlaceHolderContent(L"Reply"))
+    .AddButton(Button(L"Send")
+        .AddArgument(L"action", L"textReply")
+        .SetButtonStyle(ButtonStyle::Success))
+    .Show();
+```
+
+Example result:
+![Button Success Example](ButtonSuccessExample.png)
+
+The ToolTip is useful for buttons with icons and no content. Users can hover over the button with
+the cursor and a text will display describing what the button does.
+
+```c#
+AppNotificationContent()
+    .AddText(Text(L"Content"))
+    .AddTextBox(TextBox(L"textBox")
+        .SetPlaceHolderContent(L"Reply"))
+    .AddButton(Button(L"Send")
+       .AddArgument(L"action", L"textReply")
+        .SetInputId(L"textBox")
+        .SetToolTip(L"Send")
+        .SetIconUri(LR"(Assets/Icons/message.png)"))
+    Show();
+```
+
+Example result:
+![Button ToolTip Example](ButtonToolTipExample.png)
+
+The placement attribute defines action for a user to interact with when the AppNotification is
+right-clicked. This would typically be used by developers to ask the user for setting changes.
+
+```c#
+AppNotificationContent().
+    AddText(Text(L"Content")).
+    AddTextBox(TextBox(L"textBox").
+        SetPlaceHolderContent(L"Reply")).
+    AddButton(Button(L"Modify app settings").
+        SetInputId(L"textBox").
+        SetIsContextMenuPlacement(true))
+    .Show();
+```
+
+Example result:
+![Button ContextMenu Example](ButtonContextMenuExample.png)
+
+# Image
+
+The Image component API sets up the xml for the \<image\> element. Images are visual elements that
+are used to enhance an AppNotification.
+
+The image source can be pulled using one of these protocol handlers:
+
+-   http:// or https:// (A web-based image)
+-   ms-appx:/// (An image included in the app package)
+-   ms-appdata:///local/ (An image saved to local storage)
+-   file:/// (A local image)
+
+**WinAppSDK 1.2 \<image\> Schema**:
+
+```c#
+<image src = string
+       alt? = string
+       placement? = "appLogoOverride" | "hero"
+       hint-crop? = "circle" />
+```
+
+These attributes are abstracted with the Button component APIs:
+
+-   Image(String imageSrc)
+-   SetAlternateText(String alternateText)
+-   SetImagePlacement(ImagePlacement placement)
+-   SetUsesCircleCrop(Boolean usesCircleCrop)
+
+Images can be displayed in three different ways:
+
+## Inline
+
+A full-width inline-image that appears when you expand the AppNotification.
+
+```c#
+AppNotificationContent()
+    .AddText(Text(L"Below is an inline image"))
+    .AddImage(Image(LR"(https://unsplash.it/360/202?image=883)")
+        .SetImagePlacement(ImagePlacement::Inline)
+        .SetAlternateText(L"Image with nature"))
+    .Show();
+```
+
+Example result:
+![Inline Image Example](InlineImageExample.png)
+
+## AppLogoOverride
+
+Overrides the AppNotification app logo with a custom image.
```

Comment:
It doesn't override the app logo in the latest builds. I'd replace this description with "An image that is left-aligned with notification text"